cluster/gce/config-test.sh

```diff
@@ -377,6 +377,7 @@ ENABLE_LEGACY_ABAC="${ENABLE_LEGACY_ABAC:-false}" # true, false
 
 # Enable a simple "AdvancedAuditing" setup for testing.
 ENABLE_APISERVER_ADVANCED_AUDIT="${ENABLE_APISERVER_ADVANCED_AUDIT:-true}" # true, false
+ADVANCED_AUDIT_LOG_MODE="${ADVANCED_AUDIT_LOG_MODE:-batch}" # batch, blocking
```

Comment:
Do you need to set this in config-default.sh too?

Reply:
I think I'd prefer to leave the non-test default as blocking until we fix an issue that could lead to an out-of-order log.